pages/api/service/index.ts

```diff
@@ -5,97 +5,226 @@ import RealEstate from '@common/modules/models/RealEstate'
 import Service from '@common/modules/models/Service'
 import start, { Data } from '@pages/api/api.config'
 import { getCurrentUser } from '@utils/getCurrentUser'
+import { FilterQuery } from 'mongoose'
 import type { NextApiRequest, NextApiResponse } from 'next'
+import { dateToYear } from '@common/assets/features/formatDate'
+import { uniq } from 'lodash'
+import { getFormattedDate } from '@utils/helpers'
 
 start()
 
+export interface IFilter {
+  value: any
+  lable: string
+}
```

Comment:
This can be a global interface. Create new `common/modules/models/Filter.ts` file and move this code in it
Also, better to use `text` instead of `label` - in this case filter components may be used as `<Select />` options in future